Invocation_BlockingTest.async_whenMultipleAndThenOnSameFuture

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 10
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:764)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:757)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:749)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_BlockingTest.async_whenMultipleAndThenOnSameFuture(Invocation_BlockingTest.java:404)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-OracleJDK1.6/com.hazelcast$hazelcast/890/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_BlockingTest/async_whenMultipleAndThenOnSameFuture/

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 10
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:764)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:757)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:749)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_BlockingTest.async_whenMultipleAndThenOnSameFuture(Invocation_BlockingTest.java:404)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x/com.hazelcast$hazelcast/5093/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_BlockingTest/async_whenMultipleAndThenOnSameFuture/

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 10
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:764)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:757)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:749)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_BlockingTest.async_whenMultipleAndThenOnSameFuture(Invocation_BlockingTest.java:404)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.8/com.hazelcast$hazelcast/223/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_BlockingTest/async_whenMultipleAndThenOnSameFuture/

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 10
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:791)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:784)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:776)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation_BlockingTest.async_whenMultipleAndThenOnSameFuture(Invocation_BlockingTest.java:404)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-OracleJDK1.6/com.hazelcast$hazelcast/942/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/Invocation_BlockingTest/async_whenMultipleAndThenOnSameFuture/

--- a/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_BlockingTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_BlockingTest.java
@@ -334,7 +334,7 @@ public class Invocation_BlockingTest extends HazelcastTestSupport {
      */
     @Test
     public void async_whenMultipleAndThenOnSameFuture() throws Exception {
-        int callTimeout = 1000;
+        int callTimeout = 5000;
         Config config = new Config().setProperty(OPERATION_CALL_TIMEOUT_MILLIS.getName(), "" + callTimeout);
 
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);

